config/locales/oc.yml

```diff
@@ -290,263 +290,263 @@ ca:
-              en marc dera Organizacion Mondiau deth Comerç a d’includir un tracte
-              especiau e diferenciat, valable e efectiu entàs païsi en desvolopament e es
+            objective6: Entath 2020, proïbir cèrtes fomes de subvencions ara pesca que 
+              contribusissen ara capacitat de suberpesca e ara suberespleita pesquèra, 
+              eliminar es subvencions que contribusissen ara pesca illegau, non declarada 
+              e non reglamentada e abstier-se d’introdusir naues subvencions d’aguest 
+              biais, en tot reconéisher qu’era negociacion sus es subvencions ara pesca 
+              en marc dera Organizacion Mondiau deth Comerç a d’includir un tracte 
+              especiau e diferenciat, valable e efectiu entàs païsi en desvolopament e es 
               païsi mens auançadi.
-            objective7: Entath 2030, aumentar es beneficis economics qu’es petiti
-              estats insulars en desvolopament e es païsi mens auançadi receben der
-              us sostenible des recorsi marins, en particular mejançant era gestion
+            objective7: Entath 2030, aumentar es beneficis economics qu’es petiti 
+              estats insulars en desvolopament e es païsi mens auançadi receben der 
+              us sostenible des recorsi marins, en particular mejançant era gestion 
               sostenibla dera pesca, era aqüicultura e eth torisme.
-            objective8: Aumentar es coneishements scientifics, desvolopar era
-              capacitat de recerca e transferir era tecnologia marina, en tot tier
-              en compde es critèris e es directritzes entara transferéncia de
-              tecnologia marina dera Comission Oceanografica Intergovernamentau,
-              a fin de milhorar era salut des oceans e potenciar era contribucion
-              dera biodiversitat marina ath desvolopament des païsi en desvolopament,
-              en particular es petiti estats insulars en desvolopament e es païsi
-              mens auançadi.
-            objective9: Facilitar er accès des pescaires artesanaus en petita escala as
-              recorsi marins e as mercats.
-            subtitle: Vida submarina
+            objective8: Aumentar es coneishements scientifics, desvolopar era 
+              capacitat de recerca e transferir era tecnologia marina, en tot tier 
+              en compde es critèris e es directritzes entara transferéncia de 
+              tecnologia marina dera Comission Oceanografica Intergovernamentau, 
+              a fin de milhorar era salut des oceans e potenciar era contribucion 
+              dera biodiversitat marina ath desvolopament des païsi en desvolopament, 
+              en particular es petiti estats insulars en desvolopament e es païsi 
+              mens auançadi. 
+            objective9: Facilitar er accès des pescaires artesanaus en petita escala as 
+            recorsi marins e as mercats.
```

Comment:
Hi @spuigmitjadxc 
a tab is missing here please, line 466
If we validate .yml file's format, turns out an error
![image](https://user-images.githubusercontent.com/9702463/106103104-b5c01e00-6140-11eb-8150-234bc131717a.png)
Thanks! :pray: